deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -110,8 +110,8 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
 
     private lazy var muteItem: UIBarButtonItem = {
         let imageView = UIImageView()
-        imageView.tintColor = DcColors.defaultTextColor
-        imageView.image =  UIImage(named: "volume_off")?.withRenderingMode(.alwaysTemplate)
+        imageView.tintColor = DcColors.middleGray
```

Comment:
The idea of using a `customView` is probably due to some margins and constraints, right?

Reply:
good question, i just don't know, i changed only the image.

maybe it is as the items should not be buttons but more state images